packages/features/tasker/tasks/triggerNoShow/common.ts

```diff
@@ -0,0 +1,140 @@
+import dayjs from "@calcom/dayjs";
+import { sendGenericWebhookPayload } from "@calcom/features/webhooks/lib/sendPayload";
+import { fetcher } from "@calcom/lib/dailyApiFetcher";
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import type { TimeUnit } from "@calcom/prisma/enums";
+import { BookingStatus, WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+import { getBooking } from "./getBooking";
+import type { TWebhook, TTriggerNoShowPayloadSchema } from "./schema";
+import { triggerNoShowPayloadSchema, ZSendNoShowWebhookPayloadSchema } from "./schema";
+
+type Host = {
+  id: number;
+  email: string;
+};
+
+type Booking = Awaited<ReturnType<typeof getBooking>>;
+type Webhook = TWebhook;
+export type Participants = TTriggerNoShowPayloadSchema["data"][number]["participants"];
+
+export const getMeetingSessionsFromRoomName = async (roomName: string) => {
+  return fetcher(`/meetings?room=${roomName}`).then(triggerNoShowPayloadSchema.parse);
+};
+
```

Comment:
This fetches all the meeting sessions where 1 meeting session ends when everyone leaves the meeting.